### PR TITLE
Review Changes

### PR DESCRIPTION
**Pull request by rushikesh3299**
1. MenuBar update for p1 remainig components
2. Ifrmaes code snippet width adjustments
3. code indentation issues
4. Added Favicon

### CODE REVIEW

**Review by Bharatrathod-81 — APPROVED** [review]
perfect code

File: Components/Button/button.html [hunk 1]
```diff
@@ -20,7 +24,8 @@
             <div class="nav-right">
                 <a href="https://github.com/rushikesh3299"><i class="fab fa-github"></i></a>
                 <a href="https://twitter.com/rushikesh3299"><i class="fab fa-twitter"></i></a>
-                <a href="https://www.linkedin.com/in/rushikesh-chougule-621854192/"><i class="fab fa-linkedin-in"></i></a>
+                <a href="https://www.linkedin.com/in/rushikesh-chougule-621854192/"><i
```
Comment (ashishnnnnn) [inline Components/Button/button.html:27]: You can use target="_blank" for the external links

**Review by JantuDeb — APPROVED** [review]
Overall codes looks fine 👍

**Review by ashishnnnnn — COMMENTED** [review]
Overall it looked good and modular..

**Pull request merged**